### ISSUE TITLE
Extreme graphical glitch on old thinkpad T60. Text wraps around the screen, not readable.

### ISSUE THREAD

**Issue by ianniculescu** [swaywm/sway#7272]
### Please read the following before submitting:
- Please do NOT submit bug reports for questions. Ask questions on IRC at #sway on Libera Chat.
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway.
- Please do NOT submit issues for information from the github wiki. The github wiki is community maintained and therefore may contain outdated information, scripts that don't work or obsolete workarounds.
  If you fix a script or find outdated information, don't hesitate to adjust the wiki page.

### Please fill out the following:
- **Sway Version:**
  - `swaymsg -t get_version` or `sway -v`
  - sway version 1.7

- **Debug Log:**
  - Run `sway -d 2> ~/sway.log` from a TTY and upload it to a pastebin, such as gist.github.com.
  - This will record information about sway's activity. Please try to keep the reproduction as brief as possible and exit sway.
  - Attach the **full** file, do not truncate it.
  - gist <script src="https://gist.github.com/ianniculescu/a06284ad6072fd1824beaf77e7d29eec.js"></script>

- **Configuration File:**
  - Please try to produce with the default configuration.
  - If you cannot reproduce with the default configuration, please try to find the minimal configuration to reproduce.
  - Upload the config to a pastebin such as gist.github.com.

- **Stack Trace:**
  - This is only needed if sway crashes.
  - If you use systemd, you should be able to open the coredump of the most recent crash with gdb with
    `coredumpctl gdb sway` and then `bt full` to obtain the stack trace.
  - If the lines mentioning sway or wlroots have `??` for the location, your binaries were built without debug symbols. Please compile both sway and wlroots from source and try to reproduce.

- **Description:**
  - The steps you took in plain English to reproduce the problem.
  - Bought an old thinkpad T60 with resolution 1400*1050, and GPU AMD ATI Mobility Radeon X1400 to try Arch Linux for the first time. Used the archinstall script do the work of installing sway with the right open-source ATI drivers and all the needed packages, also install xorg-xwayland on my own. (altought I have the same issue when installing arch manually.)
 Can't run sway before also installing polkit. 
Finally when I run sway I can use mod + return and run commands in the terminal without issue. The PROBLEM is that everything wraps around the screen, text is unreadable, the curser is a 
bunch of dots on multple lines, and can't figure out what to do. Provided pictures below describing the problem.
![launching sway](https://user-images.githubusercontent.com/90414834/202854651-aced2f07-8036-4232-bb20-64a0d1f8db15.jpeg)
![sway terminal](https://user-images.githubusercontent.com/90414834/202854694-d41286e5-90c6-4e63-807d-2a83b2885248.jpeg)

short video showing the problem
https://user-images.githubusercontent.com/90414834/202855137-e3492d81-52cb-4b3f-bca3-fb9489950978.mp4

here pictures with various steps before I could run sway including neofetch, polkit ...
![ati](https://user-images.githubusercontent.com/90414834/202854744-e8a15196-625a-49f7-9069-56ecdb9a60a6.jpeg)
![neofetch](https://user-images.githubusercontent.com/90414834/202854759-9a112486-7550-444e-a1a1-fd05f99b6f1d.jpeg)
![sway without polkit](https://user-images.githubusercontent.com/90414834/202854771-4d92f62e-65f2-4122-8270-9b
![sudo sway](https://user-images.githubusercontent.com/90414834/202854788-9c770f88-2444-4f58-bd26-eda8ebfa3973.jpeg)
![polkit status](https://user-images.githubusercontent.com/90414834/202854811-c258e81d-02d5-48e0-b13d-3b3951dcef51.jpeg)
![sway --debug](https://user-images.githubusercontent.com/90414834/202855286-62c265c1-21ec-44b3-854a-9ea9ee17371d.jpeg)

Thank you for your work on sway !

**Comment by emersion** [comment]
Your bug report is missing debug logs. Please add a link to the full debug log file.

    sway -d >sway.log 2>&1

**Comment by emersion** [comment]
It's not a debug log, it's a regular log. Also it's a screenshot.

**Comment by emersion** [comment]
Nope. The Vulkan renderer is not supported on older hardware and is still experimental.

**Comment by bl4ckb0ne** [comment]
Fwiw using the pixman renderer would be more suited if the card is too old to have an opengl driver.

**Comment by bestuzheff** [comment]
Please! Fix this problem!
